### ISSUE TITLE
Add Possibility to Set Project / Task as Default

### ISSUE THREAD

**Issue by KornKalle** [kimai/kimai#4528]
### Describe the problem

We are planning to use Kimai with ~50 employees. Most of them are working for only one project, doing the same "task" every day.
Some of our employees are challenged with using web interfaces and it improves their lifes when they need to interact as little as possible with those interfaces.

### Describe the solution you'd like

I would appreciate it if there was an option to set a Project and Task as Default. Either globally, per Team or per User.
For our needs it would be enough to configure at per Team or even globally.

**Comment by MartinRyx** [comment]
Personally I only "need" a default project, since most of my time is registered on one single project, but a lot of different tasks. On the other hand, a default task on every project could be quite nice. Then I could default to an "uncategorized" task when switching tasks and then I could change later if I want. Great idea anyhow.
And in my opinion the setting should be per user

**Comment by oliverpool** [comment]
I just looked for a similar button, which would allow me to `switch task`:
- if a task is already started, end this task (otherwise goto to the next step directly)
- prefill the "new task popup" with the same info as the last task (so that i can adjust it quickly)

---

Thinking more about it, maybe the `Restart one of your last activities` popup can add a button to adjust some things, before restarting the task.
So in addition to `restart_timesheet`, a link to `Start as-is` (current behavior) or `Edit before starting` (which acts like `Create a copy`):

![image](https://github.com/user-attachments/assets/8a9f4717-c509-447d-9ebd-5b49b9317532)

@kevinpapst I can try to craft a PR if you think that this would be a good addition.

Note to myself: likely need to edit the following file
https://github.com/kimai/kimai/blob/dcc52f1a9510be3dfe954255f2f87e4393a3b0a8/templates/favorite/index.html.twig#L17

**Comment by oliverpool** [comment]
Actually the first post seems to be a duplicate of #403 

And for my reply, I realized that the `Restart activity` button always set an empty description, so I just use it and then adjust the description afterwards.

So I guess this issue can be closed?

**Comment by kevinpapst** [comment]
This can also be achieved with the Custom content plugin https://www.kimai.org/store/custom-content-bundle.html
and some javascript. Examples can be found at: https://www.kimai.org/documentation/plugin-custom-content.html
